lib/app/routes/languages.rb

```diff
@@ -15,6 +15,7 @@ class Languages < Core
 
 				locals = {
           problems: Presenters::Special::Problems.new(track_id).track_problems,
+					docs: Presenters::Docs.new(track_id).fetch_docs,
 		      language: Language.of(track_id),
           slug: track_id,
```

Comment:
I really don't know what happened there. Sorry. It might of been when I was squashing things. Everything was all loopy today.